cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -235,6 +235,27 @@ def test_monthly_charges(appliance, has_no_providers_modscope, setup_provider, c
         assert monthly_charges != '$0'
 
 
+@pytest.mark.long_running
+@pytest.mark.parametrize('context', [ViaSSUI])
+@pytest.mark.parametrize('order_service', [{'vm_count': '1'}, {'vm_count': '2'}], indirect=True,
+                ids=['vm-count1', 'vm-count2'])
+def test_service_chargeback_multiple_vms(appliance, has_no_providers_modscope, setup_provider,
+        context, order_service, run_service_chargeback_report):
+    """Tests chargeback data for a service with multiple VMs
+
+    Polarion:
+        assignee: nachandr
+        casecomponent: SelfServiceUI
+        caseimportance: high
+        initialEstimate: 1/2h
+    """
+    with appliance.context.use(context):
+        dashboard = Dashboard(appliance)
+        monthly_charges = dashboard.monthly_charges()
+        logger.info('Monthly charges is {}'.format(monthly_charges))
```

Comment:
When logging, please use `logger.info('The string %s', the_var)` syntax

Reply:
Done